### ISSUE TITLE
catphan results: add low_contrast result (pass/no pass) in default results

### ISSUE THREAD

**Issue by AlainSottiaux** [jrkerns/pylinac#293]
(V2.3.1)
Catphan, results()
Number of low contrast region "seen" is shown, but test against tolerance (pass/no pass) is not.
This value is already available in CTP515 properties (overall_passed), and so could already be manually added to any analysis output.

**Comment by jrkerns** [comment]
Indeed. You are free to make your own custom `.results()` method that contains the information you'd like.
